rules/Php80/Rector/ClassMethod/AddParamBasedOnParentClassMethodRector.php

```diff
@@ -211,7 +211,7 @@ function (Node $subNode) use ($parentClassMethodParam): bool {
             $node->params[$key] = new Param(
                 new Variable($paramName),
                 $paramDefault,
-                $parentClassMethodParam->type,
+                null,
```

Comment:
Settings original null on `$parentClassMethodParam` might help

Reply:
I tried that, not work, got same error `assert($startPos >= 0 && $endPos >= 0)`

Reply:
@TomasVotruba fixed with print the param type https://github.com/rectorphp/rector-src/pull/1455/commits/b6d8fe72f66944394675981fbfa2343687760ecf

Reply:
Event better :+1: :slightly_smiling_face:  thank you!

Reply:
I'm not sure priting is the right way, as printing should be handled outside Rector rules in own step.

Reply:
Ok, updated with `->getName()` https://github.com/rectorphp/rector-src/pull/1455/commits/773f047c2da7164f5e14b1722dc25f98729007c3